packaging/install_rpm.spec

```diff
@@ -16,22 +16,17 @@ URL:            https://github.com/cloudify-cosmo/cloudify-manager-install
 Vendor:         Cloudify Platform Ltd.
 Packager:       Cloudify Platform Ltd.
 
-BuildRequires:  python >= 2.7, python-virtualenv
-Requires:       python >= 2.7
+BuildRequires:  python3 >= 3.6, python3-devel >= 3.6, python3-virtualenv
+Requires:       python3 >= 3.6
 
 %description
 Cloudify Manager installer.
 
 %build
 mkdir -p $(dirname %_venv)
-virtualenv %_venv
+virtualenv -p python3 %_venv
```

Comment:
whenever possible prefer the stdlib venv instead of this one:
`python3 -m venv %_venv`
(also allows to drop `BuildRequires: python3-virtualenv`)